bin/generate_schema

```diff
@@ -11,22 +11,20 @@ fi
 
 echo "Generating Swagger schema"
 src/manage.py generate_swagger \
-    ./src/swagger2.0.json \
+    --output_file ./src/openapi.yaml \
```

Comment:
Dit is staat niet echt heel consistent qua naamgeving, ik zou hier `--output-file` van maken